frontend/homework/3/tipCalculator.js

```diff
@@ -0,0 +1,26 @@
+const r = [140, 45, 280];
+
+function tipCalculator(...args) {
```

Comment:
The function name must be generic as you are using this function for both total and tip calculations